pydantic_ai_slim/pydantic_ai/ui/web/app.py

```diff
@@ -0,0 +1,57 @@
+"""Factory function for creating a web chat app for a Pydantic AI agent."""
+
+from __future__ import annotations
+
+from typing import TypeVar
+
+import fastapi
+import httpx
+from fastapi import Request
+from fastapi.responses import HTMLResponse
+
+from pydantic_ai import Agent
+from pydantic_ai.builtin_tools import AbstractBuiltinTool
+
+from .agent_options import AIModel, BuiltinTool
+from .api import create_api_router
+
+CDN_URL = 'https://cdn.jsdelivr.net/npm/@pydantic/ai-chat-ui/dist/index.html'
```

Comment:
I think we should pin a specific version, in case we make backward-incompatible changes at some point